easybuild/easyconfigs/c/CharLS/CharLS-2.0.0-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'CMakeMake'
+
+name = 'CharLS'
+version = '2.0.0'
+
+homepage = 'https://github.com/team-charls/charls'
+description = """CharLS is a C++ implementation of the JPEG-LS standard for lossless and near-lossless image
+compression and decompression. JPEG-LS is a low-complexity image compression standard that matches JPEG 2000
+compression ratios."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+toolchainopts = {'pic': True}
+
+source_urls = ['https://github.com/team-charls/charls/archive/']
+sources = ['%(version)s.tar.gz']
+checksums = ['528c6a3cc168a44e73f2890d8f4a35104a54d752eba3d6a643f050b72dd67cfa']
+
+builddependencies = [
+    # use same binutils version that was used when building GCC toolchain
+    ('binutils', '2.30', '', True),
```

Comment:
@crubb No need to use dummy binutils here, please use ('binutils', '2.30') instead